Sources/TestSupport/SwiftPMProduct.swift

```diff
@@ -84,4 +89,16 @@ public enum SwiftPMProduct {
             throw error
         }
     }
+
+    public static func packagePath(for packageName: String, packageRoot: AbsolutePath) throws -> AbsolutePath {
+        // FIXME: This uses package tool right now, and should be replaced by --get-package-path
+        // whenever we design that.
+        try POSIX.chdir(packageRoot.asString)
+        let packageTool = SwiftPackageTool()
```

Comment:
We can pass in the command line arguments with added `--chdir` flag but will have to modify all the tools